modules/37_industry/subsectors/sets.gms

```diff
@@ -198,6 +198,13 @@ $endif.process_based_steel
                       feh2_otherInd, fehe_otherInd, feelhth_otherInd)
   /
 
+  in_chemicals_37(all_in)   "chemicals FE that can provide feedstocks"
```

Comment:
Does it make sense to rename this set to include something about the feedstocks in its name? 
The was it is now, the name `in_chemicals_37` does imply a more general set than its intention.

Reply:
alrighty, now it's called `in_chemicals_feedstock_37`